Felienne/hedy
# [FEATURE] Auto scrolls output view down with output

**Description**
In this PR we make a small improvement to the output view, automatically scrolling down with the loading content. Especially since we introduced the `sleep` command this has become more relevant. Now the view is automatically scrolled to the actual content. However, we might want to make a toggle for this in the future?

**Fixes**
This PR fixes #2464.

**How to test**
Compile the Typescript locally as I seem to have some issues currently with compiling. Therefore for an unknown reason the current app bundle doesn't contain the syntax highlighting. Compile once yourself to fix this. To actual test the PR itself:

Navigate to any level with the `repeat` command (for example level 7). Repeat a random program enough times so the output view is not large enough and verify that the window automatically scrolls down.
